Wabbajack.Common/WorkQueue.cs

```diff
@@ -27,64 +30,139 @@ public class WorkQueue : IDisposable
         private readonly Subject<CPUStatus> _Status = new Subject<CPUStatus>();
         public IObservable<CPUStatus> Status => _Status;
 
-        public List<Thread> Threads { get; private set; }
+        private int _nextCpuID = 1; // Start at 1, as 0 is "Unassigned"
```

Comment:
Not that it matters too much, but figured I'd point out that the CPU ID is now not `0-(num cores)`, but 0 and increasing to any value (which may be higher than the current number of active cores).  The logic was a bit simpler this way, and I don't think anyone was leveraging CPU ID in a way that required it to be within `0-(num cores)`.